gradle/libs.versions.toml

```diff
@@ -35,6 +35,8 @@ org-jetbrains-kotlin-kapt = { id = "org.jetbrains.kotlin.kapt", version = "1.9.2
 [versions]
 
 kotlin = "1.9.23"
+##  ⬆ = "1.9.24"
+##  ⬆ = "2.0.0"
```

Comment:
is this a skipped update?

Reply:
its problematic because detekt doesn't support kotlin >1.9.23